index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <nav>
+        <div id="nav-container">
+            <div class="nav-item">
+                <h1>Welcome</h1>
+            </div>
+            <div class="nav-item nav-item2"><img src="img/Icon - Menu.png" alt="Menu Icon"></div>
+        </div>
+    </nav>
+
+    <section>
+        <div id="home">
+            <div class="welcome-text first-welcome-text">
+                <p>Hey there. I’m Raphael
+                    <div id="color-green">
```

Comment:
[ ] we use a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`, This way will be semantically correct and the screen readers and other technologies will thank you.kindly fix that 🤓